File: climbing_stairs/climbing_stairs.ts
const climbStairs = (n: number): number => {
  let left = 1;
  let right = 1;
  let start = 0;
  while (start < n - 1) {
    const temp = left;
    left = left + right;
    right = temp;
    start++;
  }
  return left;
};

File: climbing_stairs/Climbing_stairs.java
package climbing_stairs;

public class Climbing_stairs {
    public int climbStairs(int n) {
        int left = 1;
        int right = 1;
        for (var i = 0; i < n - 1; i++) {
            int temp = left;
            left = left + right;
            right = temp;
        }
        return left;
    }
}
